pyTigerGraph/pyTigerGraphBase.py

```diff
@@ -124,6 +126,14 @@ def __init__(self, host: str = "http://127.0.0.1", graphname: str = "MyGraph",
         else:
             self.authHeader = {"Authorization": "Basic {0}".format(self.base64_credential)}
 
+        # If JWT token is provided, set authMode to "token", and overwrite authMode = "pwd" for GSQL authentication as well if version is newer than 4.1.0
+        if jwtToken:
+            dbVersion = self.getVer()
```

Comment:
This version check will fail since version relies on token auth, and the auth header has not yet been defined.